### PR TITLE
Hotfix 0.10.1

### PR DESCRIPTION
**Pull request by oleurud**
Release a new 0.10.1 version with bug fixes: 
- [x] https://github.com/CartoDB/cartoframes/issues/732
- [x] https://github.com/CartoDB/cartoframes/issues/754
- [x] https://github.com/CartoDB/cartoframes/issues/734
- [x] version 0.10.1
- [ ] publish

**NOTE**: After that, we should ensure our fixes in the `develop` branch

### CODE REVIEW

**Comment by Jesus89** [comment]
Hello! I have merged the rest of bugfix PRs.

@andy-esch when you are ready, please take a look at this PR to prepare the next patch release.

File: NEWS.rst [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+
+Release 2019-06-xx
+
+Updates
+
+- Fix schema not always properly set in write operations (#734)
+- Fix error in Dataset.upload related with array data (#754)
+- Fix Dataset.download error when reading boolean column with nulls (#732)
+
+Changelog
```
Comment (andy-esch) [inline NEWS.rst]: Duplicate title

File: README.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@ CARTOframes
 .. image:: https://coveralls.io/repos/github/CartoDB/cartoframes/badge.svg?branch=master
     :target: https://coveralls.io/github/CartoDB/cartoframes?branch=master
 .. image:: https://mybinder.org/badge_logo.svg
-    :target: https://mybinder.org/v2/gh/cartodb/cartoframes/v0.9.2?filepath=examples
+    :target: https://mybinder.org/v2/gh/cartodb/cartoframes/v0.10.1?filepath=examples
```
Comment (andy-esch) [inline README.rst]: ❤️ thanks for getting this

**Comment by Jesus89** [comment]
OK @andy-esch. Feel free to merge this PR in `master` and then release the version :muscle: 

We will merge this branch in `develop`.

**Comment by andy-esch** [comment]
Cool, I'll merge once tests pass and do the release. Thanks for all the work @oleurud and @Jesus89!!

**Pull request merged**